### ISSUE TITLE
colab test getting stuck

### ISSUE THREAD

**Issue by edublancas** [ploomber/projects#36]
the colab test is getting stuck, it keeps running but it never finishes - no changes have been made recently. I comment it out so it doesn't break the CI

https://github.com/ploomber/projects/blob/16fcd6815c8c216e96b06fcd289cd3f317361215/.github/workflows/ci.yml#L153

**Comment by idomic** [comment]
Yeah there's an issue with it I think regardless to the nb convert

**Comment by edublancas** [comment]
can you fix it?

**Comment by edublancas** [comment]
I tried running the actual colab (not the CI version) and it also gets stuck:

![image](https://user-images.githubusercontent.com/989250/168486267-1523544c-f3dc-485b-8801-ab1b5b5901fa.png)


maybe we should remove it?

**Comment by idomic** [comment]
I'll take a look, seems like they changed something on their end.
